src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -296,10 +296,14 @@ private function getDocBlockIndex(Tokens $tokens, $index)
      */
     private function hasTestPrefix($functionName)
     {
-        if (!$this->startsWith('test', $functionName) || strlen($functionName) < 5) {
+        if (!$this->startsWith('test', $functionName)) {
             return false;
         }
 
+        if ('test' === $functionName) {
```

Comment:
@keradus @SpacePossum 

Is this better?

Reply:
:+1: it is more explicit and faster, does this solution work for `TesT` etc. as well?

Reply:
I can adjust!

I think, though, that for PHPUnit it needs to be `test` (case-sensitive) to qualify as test.

Reply:
Also see [`PhpUnitTestAnnotationFixer::startsWith()`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.9.1/src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php#L257-L262), where we compare case-sensitively.

Reply:
ah that makes sense! never mind me :)